src/circuits/joinsplit.tcc

```diff
@@ -619,11 +618,17 @@ public:
         // < FieldT::capacity()
         nb_elements += 1;
 
-        // We allocate 2 field elements to pack the value of h_sig
-        nb_elements += 2;
+        // h_sig is represented by 2 field element (if we consider a digest_len
```

Comment:
Why is this showing up in this PR? This is related to the transaction malleability fix which has already been merged to develop

Reply:
Any news on that @rrtoledo?

Reply:
I wanted the comments to be consistent, the parenthesis was missing.